packages/client/validation/makeUserServerSchema.js

```diff
@@ -1,13 +1,9 @@
-import {urlRegex} from './regex'
+import { url } from 'parabol-client/validation/templates'
 import legitify from './legitify'
 
 export default function makeUserServerSchema() {
   return legitify({
-    picture: (value) =>
-      value
-        .trim()
-        .matches(urlRegex, 'that picture url doesn’t look quite right')
```

Comment:
this `urlRegex` wasn't recognizing a url with `localhost` in it as a proper url. based on my reading, it seems using a regex for url validation is error prone. several people recommended using a built-in URL parser for validation, which seems a simpler and more robust solution.